packages/toolpad-app/src/toolpadDataSources/function/client.tsx

```diff
@@ -141,7 +141,7 @@ function QueryEditor({
 
     const content = `
       interface ${EVENT_INTERFACE_IDENTIFIER} {       
-        params: {
+        parameters: {
```

Comment:
Something like this is what I meant above. But to be honest, I'm fine with removing it as well.

```suggestion
        @deprecated Use parameters instead
        params: {
          ${paramsMembers}
        }
        parameters: {
```

Reply:
Thanks! I just had to wrap it with comment, otherwise it didn't work
```
/** @deprecated Use parameters instead */
params: {
```

<img width="557" alt="image" src="https://user-images.githubusercontent.com/437214/194077761-73b45015-3cef-451f-ac41-231ed416d5bb.png">

Reply:
Ah right, if we go this route, maybe we should also add something like "will be removed in Toolpad v1" or so?
(and to the comment down below something like `TODO: remove after v1`?)